velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -144,5 +145,91 @@ TEST_F(ContainerRowSerdeTest, nested) {
   testRoundTrip(nestedArray);
 }
 
+TEST_F(ContainerRowSerdeTest, compareNullsInRowVector) {
+  std::optional<std::optional<int>> x =
+      std::make_optional<std::optional<int>>(std::nullopt);
+  auto data = makeRowVector({
+      makeNullableFlatVector<int32_t>({
+          1,
+          2,
+          3,
+      }),
+      makeFlatVector<std::string>({
+          "a",
+          "",
+          "Long test sentence ......",
+      }),
+  });
+
+  auto position = serialize(data);
+  ByteStream out;
+  HashStringAllocator::prepareRead(position.header, out);
+
+  auto someNulls = makeFlatVector<int32_t>(
+      10, [](auto row) { return row; }, [](auto row) { return row % 2 == 0; });
+  auto rowVector = makeRowVector({someNulls});
+  SelectivityVector rows(rowVector->size());
+  DecodedVector decodedVector;
+  decodedVector.decode(*rowVector, rows);
+
+  static const CompareFlags kCompareFlags{
+      true, // nullsFirst
+      true, // ascending
+      false, // equalsOnly
+      CompareFlags::NullHandlingMode::StopAtNull};
```

Comment:
I think we need more test coverage, i.e. verify all values of NullHandlingMode and combinations with different values of equalsOnly.

Reply:
Sure, I've refactored the ut, and imporved the coverage.